NoMansLand/Assets/Scripts/Oxygen/OxygenMeter.cs

```diff
@@ -48,10 +48,18 @@ void Update()
             }
             currPlayerControls.replenishHealth = false;
         }
+        
         currPercentage = slider.value / maxTime * 100f;
         if (time >= 0f)
         {
-            time -= Time.deltaTime;
+            if (currPlayerControls.decreaseHealth)
+            {
+                time -= Time.deltaTime * 10f;
```

Comment:
I found this oxygen decrease really severe tbh, after sprinting for a few seconds my oxygen was super depleted